driver/bpf/fillers.h

```diff
@@ -1035,7 +1035,48 @@ FILLER(sys_getrlimit_setrlimit_e, true)
 	return bpf_push_u8_to_ring(data, rlimit_resource_to_scap(resource));
 }
 
-FILLER(sys_getrlimit_setrlrimit_x, true)
+FILLER(sys_getrlimit_x, true)
+{
+	unsigned long val;
+	long retval;
+	s64 cur;
+	s64 max;
+	int res;
+
+	/* Parameter 1: ret (type: PT_ERRNO) */
+	retval = bpf_syscall_get_retval(data->ctx);
+	res = bpf_push_s64_to_ring(data, retval);
+	CHECK_RES(res);
+
+	/*
+	 * Copy the user structure and extract cur and max
+	 */
+	if(retval >= 0 || data->state->tail_ctx.evt_type == PPME_SYSCALL_SETRLIMIT_X)
```

Comment:
```suggestion
	if(retval >= 0)
```

Reply:
Same in the `setrlimit` filler, now that we split them there is no more need for this `if`